docs/security/CVE-2026-0861-glibc.md

```
@@ -0,0 +1,82 @@
+# CVE-2026-0861: glibc memalign Alignment Overflow
+
+**Status:** UNFIXED upstream in Debian bookworm
+**Severity:** CRITICAL
+**Suppression Expires:** 2026-03-01
+**Monitor:** https://security-tracker.debian.org/tracker/CVE-2026-0861
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)